code/modules/mob/living/carbon/xenomorph/castes/wraith/abilities_wraith.dm

```diff
@@ -391,7 +391,9 @@ GLOBAL_LIST_INIT(wraith_banish_very_short_duration_list, typecacheof(list(
 
 /datum/action/xeno_action/activable/blink/can_use_ability(atom/A, silent = FALSE, override_flags)
 	. = ..()
```

Comment:
does nothing ?
```suggestion
```

Reply:
This has been dealt with. Review please.